src/ol/map.js

```diff
@@ -1201,6 +1201,8 @@ ol.Map.prototype.renderFrame_ = function(time) {
   var view = this.getView();
   var extent = ol.extent.createEmpty();
   /** @type {?olx.FrameState} */
```

Comment:
Minor but I'm wondering if this type cast is needed (for `var previousFrameState = this.frameState_;` )